tools/ios-versioning/index.js

```diff
@@ -110,7 +110,13 @@ function postTransforms({ versionPrefix }) {
       paths: 'RNCWKWebView.m',
       replace: /\b(_SwizzleHelperWK)\b/g,
       with: `${versionPrefix}$1`,
-    }
+    },
+    {
+      // see issue: https://github.com/expo/expo/issues/4463
+      paths: 'RNCWKWebView.m',
+      replace: /MessageHandlerName = @"ReactABI\d+_\d+_\d+NativeWebView";/,
```

Comment:
Would we able to interpolate `${versionPrefix}` here (in case we change `ReactABIXX_X_X` to something else here?